tasks/prepare_binaries.js

```diff
@@ -0,0 +1,101 @@
+const path = require('path')
+const fsExtra = require('fs-extra')
+
+const { exec } = require('pkg')
+const puppeteer = require('puppeteer')
+
+const appName = 'asciidoctor-web-pdf'
+const buildPath = 'build'
+
+// Gets chromium browser and builds binaries using pkg
+// Can specify linux/mac/win as first argument to only build one of these platforms
+// NOTE: Intended to be run from the _project root only_
+
+const builds = {
+  linux: { target: 'node10-linux-x64' },
+  mac: { target: 'node10-macos-x64' },
+  win: { target: 'node10-macos-x64', suffix: '.exe', puppeteerName: 'win64' }
+}
+
+async function createPackage (builds) {
+  for (const os in builds) {
+    console.log(`Building ${appName} for: ${os}`)
+    await exec(['.', '--target', builds[os].target, '--output', `./${buildPath}/${os}/${appName}${builds[os].suffix || ''}`])
+  }
+}
+
+async function getBrowsers (builds) {
+  const targets = Object.keys(builds).map((key) => [key, builds[key].puppeteerName || key])
+  await targets.forEach(async ([dir, os]) => {
+    const f = puppeteer.createBrowserFetcher({
+      platform: os,
+      path: path.resolve(path.join('__dirname', '../', buildPath, dir, 'chromium'))
+    })
+    await f.download(puppeteer._preferredRevision, function (downloadBytes, totalBytes) {
+      // a hacky way to show progress occasionally
+      const occasionally = Math.random()
+      if (occasionally < 0.0005) {
+        process.stdout.cursorTo(0)
+        const percent = Math.round(downloadBytes / totalBytes * 100, 0)
+        process.stdout.write(`Downloading browser for ${os.padEnd(10)} ${percent.toString().padStart(5)}%`)
+      }
+    }).then(() => {
+      console.log(`Browser downloaded/available for ${os}`)
+    })
+  })
+}
+
+function copyAssets (builds) {
+  const dirs = Object.keys(builds).map((key) => key)
+  dirs.forEach(async (dir) => {
+    console.log(`Copying MathJax / examples / css / fonts into ${dir}`)
+    const basePath = path.join(__dirname, '../')
+    const outDir = path.join(basePath, buildPath, dir)
+    const mathjaxBinaryDir = path.join(outDir, 'assets', 'mathjax')
```

Comment:
I think MathJax should be included in the binary

Reply:
Mathjax is embedded into the html and the html is loaded by Chromium which does not see the snapshotted filesystem of `pkg` so I believe it needs to be external so "executable Chrome" can see it. It wasn't originally and I couldn't get it to work. I believe this is the reason why.

Reply:
OH that's right, MathJax will try to load additional ressources (at runtime) from the file system.

I have not found been able to create a "standalone" MathJax bundle, see: https://github.com/Mogztter/asciidoctor-web-pdf/issues/316

In short, you've taken the right approach :+1:

Reply:
Can we add a comment so our future selves won't fall into this trap thinking that MathJax can be embedded into the binary (until they realized it's not possible) :wink: